extensions/sql-migration/src/api/azure.ts

```diff
@@ -307,27 +307,32 @@ export interface MigrationControllerNode {
 export interface StartDatabaseMigrationRequest {
 	location: string,
 	properties: {
-		SourceDatabaseName: string,
-		MigrationController: string,
-		BackupConfiguration: {
-			TargetLocation: {
-				StorageAccountResourceId: string,
-				AccountKey: string,
+		sqlMigrationServices: string,
```

Comment:
This is plural but is a single string value - would an array not be better? (although it's not even clear what this value is for since you already have the migrationService value below this)